Examples/GithubPaginatedSearch.swift

```diff
@@ -14,7 +14,7 @@ import RxFeedback
 fileprivate struct Repository {
     var name: String
     var url: URL
-    
+
```

Comment:
What's with all the extra empty lines / trailing spaces? Do you have the Xcode "Trim whitespace" configuration set-up ? 

Last one here:

<img width="551" alt="c3c686cd-1857-4a34-bb4d-12f7855125c0" src="https://user-images.githubusercontent.com/605076/35049852-f669e1fc-fba9-11e7-9b85-ccd09d5f7a71.png">

Reply:
Yes, I have it checked!

Reply:
@freak4pc What do you need exactly there ??